openpype/hosts/blender/plugins/create/create_pointcache.py

```diff
@@ -16,20 +16,36 @@ class CreatePointcache(openpype.hosts.blender.api.plugin.Creator):
     icon = "gears"
 
     def process(self):
-
+        """ Run the creator on Blender main thread"""
+        mti = ops.MainThreadItem(self._process)
+        ops.execute_in_main_thread(mti)
+
+    def _process(self):
+        # Get Instance Container or create it if it does not exist
+        instances = bpy.data.collections.get(AVALON_INSTANCES)
+        if not instances:
+            instances = bpy.data.collections.new(name=AVALON_INSTANCES)
+            bpy.context.scene.collection.children.link(instances)
+
+        # Create instance object
         asset = self.data["asset"]
         subset = self.data["subset"]
-        name = openpype.hosts.blender.api.plugin.asset_name(asset, subset)
-        collection = bpy.data.collections.new(name=name)
-        bpy.context.scene.collection.children.link(collection)
+        name = plugin.asset_name(asset, subset)
+        asset_group = bpy.data.objects.new(name=name, object_data=None)
+        asset_group.empty_display_type = 'SINGLE_ARROW'
+        instances.objects.link(asset_group)
```

Comment:
It seems previously this was using Collection instead of linking by Objects. Is there a reason for the switch? I'm no blender export so correct me on how these work, but is it the case that:

- Objects can only link to one object (like parenting)
- Objects can link to multiple Collections (like sets; or objectSets in maya)

If that is the case - why don't we use Collections for 'instances' because that would then solved [this comment](https://github.com/ynput/OpenPype/pull/5747#issuecomment-1761352543) since objects wouldn't need to be parented.

Reply:
@BigRoy  yes, u are pretty right
However blender completely lacks maya Grouping so we need to also ensure to have sort of parent with transforms which Collections (as maya Sets) lacks. The only way to do so is via Locators instead.

But incorporating Collections more into workflow is def a thing for multiple reasons (rendering flags, scene management etc)

Reply:
> However blender completely lacks maya Grouping so we need to also ensure to have sort of parent with transforms which Collections (as maya Sets) lacks. The only way to do so is via Locators instead.

Sorry, I don't get it. I seem to be able to link hierarchies to collections just fine. Or do you mean blender has no equivalent to a Maya "group" but you have to parent it to e.g. an "Empty" that still has some e.g. an "axis" as a locator?

It seems that Empties do indeed need to be any of:
![image](https://github.com/ynput/OpenPype/assets/2439881/8f8c7763-9337-4164-a693-e3fd67c8a16f)

Just to screenshot - this is how I'd expect the behavior as a pseudo-example:

![image](https://github.com/ynput/OpenPype/assets/2439881/a7ec9d0d-d0e3-468d-bde6-4d87026b71a6)

Which without a hierarchy would be:
![image](https://github.com/ynput/OpenPype/assets/2439881/408221c1-ef0f-4a2e-b060-85fab3a09620)

Note how both `pointcacheMain` and `modelMain` are linked to the same data. It can be contained in both just fine.

Reply:
So instead of parenting we should instead use collections linking?